crdt/causal_tree/causal_tree.go

```diff
@@ -0,0 +1,613 @@
+package causal_tree
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math"
+	"sort"
+
+	atm "github.com/crdteam/causal-tree/crdt/atom"
```

Comment:
I'd rather use the actual package names `atom` and `weft`, and have variables called `a` and `w`. You can use the compiler errors to identify where they are.

Reply:
done.

Reply:
done